examples/EventV1/EventV1.json

```diff
@@ -0,0 +1,28 @@
+{
+    "specversion" : "0.2",
+    "id": "ProjectId:98765,JobId:54321,TaskId:12345,EventId:34567",
+    "type": "https://nuventra.com/event/nae.ProcessStartRequested.EventV1",
+    "source":"https://nuventra.com/event/nae/ProcessStartRequested",
+    "datacontenttype":"application/json",
+    "dataschema":"https://nuventra.com/event/nae/ProcessStartRequested/DataV1",
+    "subject":"nae.pknca.ev",
+    "time":"2021-3-17T00:05:32.000Z",
+    "data": {
+        "input": [
+            {
+                "id":"ProjectId:98765,JobId:54321,ResourceId:76543",
+                "type": "nae.pknca.input.config",
+                "source": "https://nuventra.com/fileserver/",
+                "datacontenttype": "application/json",
+                "name": "config.json"
```

Comment:
Is there an example somewhere of what the configuration JSON data should look like?

Reply:
See https://raw.githubusercontent.com/nuventra/nae/main/src/test/nae.pknca.ev.input.config.json

Reply:
That link 404's. Here: https://github.com/nuventra/nae/blob/main/src/test/nae.pknca.ev.input.config.json